jwb.txt

```diff
@@ -0,0 +1,6 @@
+1. Buat branch release
+2. Buat branch bug/b yang diambil dari branch release dan fix bug lalu commit
+3. Merge branch bug/b ke release
+4. Buat branch feature/a dan feature/c yang diambil dari branch release
+5. jika feature/a atau feature/c selesai, commit dan merge ke release
+6. Lakukan Pull Request ke branch master
```

Comment:
pull request from which branch?
@andrewdudu

Reply:
Pull Request dari branch release to master @andirdju